src/openapi.yaml

```diff
@@ -3109,27 +3083,187 @@ paths:
       security:
       - JWT-Claims:
         - documenten.lezen
+    put:
+      operationId: objectinformatieobject_update
```

Comment:
not sure if this was here or not? But it doesn't feel like it should be here after the relation reversal